src/task-names.ts

```diff
@@ -1,3 +1,5 @@
 export const TASK_CIRCUITS_COMPILE = "zkit:compile";
+export const TASK_CIRCUITS_COMPILE_SHALLOW = "zkit:compile-shallow";
+export const TASK_CIRCUITS_SETUP = "zkit:setup";
```

Comment:
What do you think about this renaming? To make everything one-word.

```suggestion
export const TASK_CIRCUITS_MAKE = "zkit:make";
export const TASK_CIRCUITS_COMPILE = "zkit:compile";
export const TASK_CIRCUITS_SETUP = "zkit:setup";
```

Reply:
Sounds good